packages/iris-grid/src/IrisGridUtils.ts

```diff
@@ -1671,6 +1671,20 @@ class IrisGridUtils {
 
     return { groups: [...groupMap.values()], maxDepth, groupMap, parentMap };
   }
+
+  static convertValueToText(value: unknown, columnName?: string): unknown {
```

Comment:
Add unit tests and JSDocs for new utils function.
Also param name should be `columnType`, not `columnName`. And it should return a `string`.